UI/src/components/Home.vue

```diff
@@ -18,44 +17,59 @@
+                    </a></template>
+                  </i18n-t>
                 </ul>
-                <p>For a more interactive experience, please join
-                  one of the <a href="http://lists.ledgersmb.org/" target="_blank"
-                                rel="noopener noreferrer">{{ $t(`mailing lists`) }}</a></p>
 
-                <p>There's also the near-realtime chat experience in the
-                  <a href="https://app.element.io/%23/room/%23ledgersmb:matrix.org"
-                    target="_blank" rel="noopener noreferrer">matrix
-                      &quot;LedgerSMB&quot; room</a></p>
+                <i18n-t keypath="i18n.home.experience.text" tag="p" scope="global">
+                  <template #url><a :href="experience_url" target="_blank" rel="noopener noreferrer">
+                      {{ $t("i18n.home.experience.link") }}
+                  </a></template>
+                </i18n-t>
+
+                <i18n-t keypath="i18n.home.nearRealtime.text" tag="p" scope="global">
+                  <template #url><a :href="nearRealtime_url" target="_blank" rel="noopener noreferrer">
+                    {{ $t("i18n.home.nearRealtime.link") }}
+                  </a></template>
+                </i18n-t>
             </div>
             <div class="welcomed w3">
                 <h2>{{ $t(`Contributing`) }}</h2>
-                <p>{{ $t(`The project is always looking for contributions. The easiest ways to
-                  contribute to the project are:`) }}</p>
+                <p>{{ $t("i18n.home.contributions") }}</p>
                 <ul>
-                  <li>Report your bugs and problems
-                    to <a href="https://github.com/ledgersmb/LedgerSMB/issues/">the
-                      project's GitHub bug tracker</a> or
-                    <a href="https://lists.ledgersmb.org/">{{ $t(`mailing lists`) }}</a></li>
-                  <li>Help translate the software through the <a href="https://www.transifex.com/ledgersmb/ledgersmb/">LedgerSMB
-                      project Transifex web-based translation project</a></li>
-                  <li>Fix a <a href="https://github.com/ledgersmb/LedgerSMB/issues?q=is%3aopen%20is%3aissue%20label%3abite-sized%20-label%3aenhancement">{{ $t(`bug`) }}</a> and
-                    submit a &quot;Pull Request&quot;<br />
-                    When you do, please start with one marked 'bite-sized' and contact
-                    the development team through IRC and/or matrix for help getting
-                    started.</li>
+                  <i18n-t keypath="i18n.home.reportBugs.text" tag="li" scope="global">
+                    <template #url1><a :href="reportBugs_url1" target="_blank" rel="noopener noreferrer">
+                        {{ $t("i18n.home.reportBugs.link1") }}
+                    </a></template>
+                    <template #url2><a :href="reportBugs_url2" target="_blank" rel="noopener noreferrer">
```

Comment:
1. why are you using an attribute bound to an attribute in the object?
2. what does the `#url1` part in `<template #url1>` mean?

Reply:
reportBugs is defined in `en.json` as
```
            "reportBugs": {
                "link1": "the project's GitHub bug tracker",
                "link2": "mailing lists",
                "text": "Report your bugs and problems to {url1} or {url2}"
            },
```
so that the texts can be translated properly and that the `{url}` variables can be substituted for the 2 url required

Reply:
I still don't understand. I'm likely behind on the capabilities of the library we're using. Do you have a reference for me to read?

Reply:
https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting